jobserver/cohortextractor_permissions.py

```diff
@@ -0,0 +1,114 @@
+    131,  # https://jobs.opensafely.org/sle-and-covid-19/
+    132,  # https://jobs.opensafely.org/assessing-the-impact-of-covid-19-on-qof-registered-patients-and-health-groups-that-experience-inequalities/
+    133,  # https://jobs.opensafely.org/effect-of-covid-19-on-prescribing-of-dependence-forming-medicines-and-the-associated-health-utilisation/
+    134,  # https://jobs.opensafely.org/impact-of-covid-19-in-people-with-chronic-lymphocytic-leukaemia/
+    135,  # https://jobs.opensafely.org/impact-of-covid-19-on-diabetes-care-in-england/
+    136,  # https://jobs.opensafely.org/gp-appointments-during-covid/
+    137,  # https://jobs.opensafely.org/healthcare-needs-for-people-with-chronic-kidney-disease-in-the-covid-19-era/
+    138,  # https://jobs.opensafely.org/hypertension-and-blood-pressure-in-the-quality-and-outcomes-framework-qof/
+    139,  # https://jobs.opensafely.org/risk-factors-for-covid-19-disease-progression-in-immunocompromised-populations/
+    140,  # https://jobs.opensafely.org/incidence-and-management-of-gout-during-and-before-the-covid-19-pandemic/
+    141,  # https://jobs.opensafely.org/the-impact-of-covid-19-on-the-care-of-people-with-sickle-cell-disease/
+    142,  # https://jobs.opensafely.org/the-impact-of-covid-19-on-prescribing-of-antimicrobials/
+    143,  # https://jobs.opensafely.org/comparison-of-risk-factors-for-hospitalizations-and-death-from-winter-infections/
+    144,  # https://jobs.opensafely.org/covid-19-impact-on-anastrazole-use-by-post-menopausal-women-with-hormone-dependent-breast-cancer/
+    145,  # https://jobs.opensafely.org/association-between-ursodeoxycholic-acid-and-covid-19-outcomes/
+    146,  # https://jobs.opensafely.org/scarlet-fever-and-invasive-group-a-strep-cases-during-the-covid-19-pandemic/
+    147,  # https://jobs.opensafely.org/winter-pressures-in-primary-care-in-the-context-of-covid-19-recovery/
+    148,  # https://jobs.opensafely.org/the-impact-of-covid-19-on-pregnancy-treatment-pathways-and-outcomes/
+    149,  # https://jobs.opensafely.org/descriptive-cohort-analysis-of-long-covid-and-vaccination-status/
+    150,  # https://jobs.opensafely.org/evaluating-the-uk-shielding-policy-during-the-covid-19-pandemic/
+    151,  # https://jobs.opensafely.org/venous-thromboembolism-and-cardiovascular-events-in-inflammatory-rheumatic-diseases/
+    152,  # https://jobs.opensafely.org/digital-access-to-primary-care-for-older-people-during-covid/
+    153,  # https://jobs.opensafely.org/what-was-the-impact-of-covid-19-on-mortality-related-to-venous-thromboembolism-in-england/
+    154,  # https://jobs.opensafely.org/validation-of-isaric-sus-phosp-data-for-covid-related-hospital-admissions/
+    155,  # https://jobs.opensafely.org/the-effect-of-long-covid-on-quality-adjusted-life-years-using-openprompt/
+}
+
+
+def project_is_permitted_to_use_cohortextractor(project):
+    if project.slug == "opensafely-internal":
```

Comment:
I remember George telling me that slug is mutable (and possibly even user-editable) so I wonder if we should `pk` here?